palantirnet/the-build
# Tech 35 pantheon install

### Testing steps

* `composer create-project palantirnet/drupal-skeleton example dev-develop --no-interaction`
* `cd example`
* `composer require --dev "palantirnet/the-build:dev-TECH-35-pantheon-install"`
* Edit `.ddev/config.yaml` and change `name` to `tech35`
* Uncomment the lines for Pantheon, and comment out the lines for Acquia (containing `docroot`, `webserver_type`, and `mariadb_version`/`mysql_version`).
* `ddev start`
* `vendor/bin/the-build-installer`
* Use `tech35` as the project machine name.
* Use `https://tech35.ddev.site` as the Drupal URI.
* Choose `pantheon` as the target platform.
* Choose to install Drupal when prompted.
* Verify the installer completes without errors.
* You should see the following during the first install when the Drupal root is moved:
```
install > setup-web-root:

     [echo] Copying settings.ddev.php to old drupal root.
```
* Verify Drupal is accessible at https://tech35.ddev.site
* If you're able, follow the steps above to switch back to Acquia.
* Note the `settings.ddev.php` is not copied this time, because it already exists in the old Drupal root.

### Remaining tasks

- [x] Revert the change to `.circleci/config.yml`
- [x] Temporarily remove Config Ignore from `drupal-first-install` until there is a stable release that supports Drupal 10: https://github.com/palantirnet/the-build/blob/TECH-35-pantheon-install/targets/drupal.xml#L278
